core-lib/Actors.ns

```diff
@@ -151,6 +152,14 @@ class Actors usingVmMirror: vmMirror usingKernel: kernel = Value (
     )
   )
 
+  private class Timer = Value (
+    vmMirror actorSetupTimer: nil.
```

Comment:
Why is this done for every instantiation?
The code looks like it would need to run exactly once, no?

Could you just check whether stuff is initialized in `actorDo:after:`?